lib/student/studentscreen.dart

```diff
@@ -29,10 +29,10 @@ class _StudentScreenState extends State<StudentScreen> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(
-        title: Text("Student Screen"),
-        backgroundColor: Color.fromRGBO(75, 209, 160, 1),
-      ),
+      // appBar: AppBar(
```

Comment:
Do we not need this now? Should we just remove it?

Reply:
I left it as a placeholder while I was messing around with the layouts - I can remove it with next PR

Reply:
Ok.